client/main.lua

```diff
@@ -84,38 +84,65 @@ end)
 
 -- Loop
 
-local letSleep = true
-
-CreateThread(function()
-    while true do
-        Wait(1)
-        letSleep = true
-        if LocalPlayer.state.isLoggedIn and not InBank then
-            local playerPed = PlayerPedId()
-            local playerCoords = GetEntityCoords(playerPed, true)
-            for k, v in pairs(Config.BankLocations) do
-                local bankDist = #(playerCoords - v)
-                if bankDist < 3.0 then
-                    letSleep = false
-                    DrawMarker(27, v.x, v.y, v.z-0.99, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.001, 1.0001, 0.5001, 0, 25, 165, 100, false, true, 2, false, false, false, false)
-
-                    if bankDist < 1.0 then
-                        DrawText3Ds(v.x, v.y, v.z-0.25, '~g~E~w~ - Access Bank')
-                        if IsControlJustPressed(0, 38) then
-                            openAccountScreen()
+if Config.UseTarget then
+    Citizen.CreateThread(function()
+        RegisterNetEvent("banking:client:OpenAccountScreen", function()
```

Comment:
For cleaniness I recommend not registering an event within a thread.

You should also use CreateThread rather than Citizen.CreateThread to keep it consistent